README.md

```diff
@@ -240,7 +240,7 @@ rm -rf manifests
 mkdir -p manifests/setup
 
                                                # optional, but we would like to generate yaml, not json
-jsonnet -J vendor -m manifests "${1-example.jsonnet}" | xargs -I{} sh -c 'cat {} | gojsontoyaml > {}.yaml; rm -f {}' -- {}
+jsonnet -J vendor -m manifests "${1-example.jsonnet}" | xargs -I{} sh -c 'cat {} | gojsontoyaml > {}.yaml; find {} -delete' -- {}
```

Comment:
The solution that @kakkoyun suggested https://github.com/coreos/kube-prometheus/issues/515#issuecomment-621105820
 works for me 👍 But I think it needs to be verified by someone from linux side as well :D 

> -jsonnet -J vendor -m manifests "${1-example.jsonnet}" | xargs -I{} sh -c 'cat {} | gojsontoyaml > {}.yaml; find {} -delete' -- {}
+jsonnet -J vendor -m manifests "${1-example.jsonnet}" | xargs -I{} sh -c 'cat {} | gojsontoyaml > {}.yaml' -- {}
+find manifests -type f ! -name '*.yaml' -delete

Reply:
we already do this in other places, so I'm happy with this solution and it's tested on linux